### DISCUSSION TITLE
PostgreSQL - See default privileges in DBeaver

### DISCUSSION THREAD

**Discussion by ridicolos** [dbeaver/dbeaver#14788] [Q&A]
I was wondering if there is a way to see all the default-privileges which are present on a PostgreSQL-Database.

This sql-script does what I want, but maybe there is something already built-in to DBeaver to display them:

```sql
SELECT 
  nspname,         -- schema name
  defaclobjtype,   -- object type
  defaclacl        -- default access privileges
FROM pg_default_acl a JOIN pg_namespace b ON a.defaclnamespace=b.oid;
```

or with psql

```
postgres=# \ddp
```

If not, maybe this is a nice idea of a feature to implement.

**Comment by ShadelessFox** [comment]
Hello @ridicolos,

DBeaver does not support displaying default privileges.

It is indeed a nice idea to create a feature request out of it.

**Reply by ridicolos** [reply]
Thanks for the answer. I create a feature request for that. #14804
